src/robot/conf/languages.py

```diff
@@ -1179,6 +1179,7 @@ class It(Language):
     and_prefixes = ['E']
     but_prefixes = ['Ma']
     true_strings = ['Vero', 'Sì', 'On']
+    teardown_setting = 'Hậu thiết lập'
```

Comment:
This addition, and the removal of the same line from the `Vi` class seems to be a mistake. Please revert.